### ISSUE TITLE
Fundamental redirects for all big archived URLs

### ISSUE THREAD

**Issue by peterbe** [mdn/yari#3692]
In https://docs.google.com/document/d/1exNMAfz58uOifkKgJp3itrWBLwvOX2kymhpMDVoqpE8/edit we decided to redirect large trees to go to the file browser in GitHub. E.g. `/en-US/docs/Mozilla/Developer_guide` should redirect to https://github.com/mdn/archived-content/tree/main/files/en-us/mozilla/developer_guide 
There are a lot of these and it would be better to just use fundamental redirects instead of clogging up the `_redirects.txt` files.

**Comment by peterbe** [comment]
Once this lands in `main` and Lambda@Edge is deployed, let's merge https://github.com/mdn/content/pull/4681

**Comment by caugner** [comment]
Closing as won't fix, as there is only a small number of 404s on the `Developer_guide`.

FWIW These are the top 404s per calendar week:
1. `https://developer.mozilla.org/en-US/docs/Mozilla/Projects/Midas/Security_preferences` (991)
2. `https://developer.mozilla.org/en-US/docs/Mozilla/Projects/Rhino` (184)
3. `https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/AMO/Policy/Contact` (119)
4. `https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/Code_snippets/StringView` (79)
5. `https://developer.mozilla.org/en-US/Mozilla_event_reference/click` (via overapi.com) (62)
6. `https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag` (62)
7. `https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey` (42)
